core/arch/x86/mangle.c

```diff
@@ -342,7 +342,7 @@ insert_push_all_registers(dcontext_t *dcontext, clean_call_info_t *cci,
     if (cci == NULL)
         cci = &default_clean_call_info;
     if (cci->preserve_mcontext || cci->num_simd_skip != NUM_SIMD_REGS) {
-        int offs = XMM_SLOTS_SIZE + PRE_XMM_PADDING;
+        int offs = MAX_TOTAL_SIMD_SLOTS_SIZE + PRE_XMM_PADDING;
```

Comment:
Still getting hung up on MAX_: wouldn't this only deal with the ones we're actually saving now?  So if we lazily save zmm, why bother to include them here?  I.e., why be a maximum?

OK like the above comment this is just a struct offset?  I would rename to not have MAX_ in the name.

Reply:
Changed to MCTX_. I think this makes more sense.